compiler/expr_visitor_test.py

```diff
@@ -35,11 +35,13 @@ def Test(self):
   return Test
 
 
-def _MakeLiteralTest(lit):
+def _MakeLiteralTest(lit, expected=None):
   def Test(self):
     status, output = _GrumpRun('print repr({!r}),'.format(lit))
```

Comment:
Seems current test case have the same problem, for example, `_MakeLiteralTest(1E6)`, the CPython parser will parse it to `100000.0` before we run it. I added a new function `_MakeLiteralStringTest`, should we change all the current tests to it ?